datahub-classify/src/datahub_classify/infotype_predictor.py

```diff
@@ -37,6 +37,7 @@ def predict_infotypes(
     confidence_level_threshold: float,
     global_config: Dict[str, Dict],
     infotypes: Optional[List[str]] = None,
+    minimum_values_threshold: int = 50,
```

Comment:
does it also make sense to decrease the default?

Reply:
In general, if we reduce the number of values we will have less confidence on the prediction, minimum 50 values are required to have confidence about proposed info type.